tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+---
+layout: tutorial
+title: Write Your First Flutter App
+permalink: /tutorials/get-started/
+---
+
+This is a guide to creating your first Flutter app. If you know
+how to program, you can complete this tutorial. You don’t need
+previous experience with Dart or mobile programming.
+
+* TOC
+{:toc}
+
+### What are Flutter and Dart?
+
+_Flutter_ is a mobile app SDK to help developers and designers
+build modern mobile apps for iOS and Android.
+
```

Comment:
Not a request to change, but this is how our current website etc puts it "Flutter is Google's new mobile UI toolkit,
helping developers craft high-quality native experiences across mobile platforms
in record time."

Reply:
Yes, changed as per Seth. I was out of date.  :)